ci-operator/jobs/openshift/api/openshift-api-release-4.4-presubmits.yaml

```diff
@@ -24,7 +24,7 @@ presubmits:
         - --target=verify
         command:
         - ci-operator
-        image: ci-operator:latest
+        image: default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/ci-op-12jq9kyf/pipeline:ci-operator
```

Comment:
You should have been able to keep this the same if you migrated the cluster?

Reply:
this is is a manually promoted ci-op images because I did not want to break everyones builds just so I can see if my changes work